app/services/identity_linker.rb

```diff
@@ -42,6 +42,7 @@ def identity
     @identity ||= find_or_create_identity_with_costing
   end
 
+  # mattw: What does this have to do with costing?
```

Comment:
it probably use to have a costing method but I bet we removed it, maybe we should rename this